nixos/tests/installer.nix

```diff
@@ -440,6 +506,7 @@ let
             docbook_xsl_ns
             kbd.dev
             kmod.dev
+            klibc
```

Comment:
This should be dependent on `clevisTest`, no?

Reply:
Yep